normandy/recipes/tests/test_models.py

```diff
@@ -447,79 +405,11 @@ def test_recipe_revise_partial(self):
             extra_filter_expression="something !== undefined",
         )
         a2 = ActionFactory()
-        c = ChannelFactory(slug="beta")
-        recipe.revise(name="changed", action=a2, channels=[c])
+        recipe.revise(name="changed", action=a2)
         assert recipe.action == a2
         assert recipe.name == "changed"
         assert recipe.arguments == {"message": "something"}
-        assert recipe.filter_expression == (
-            "(normandy.channel in ['beta']) && " "(something !== undefined)"
-        )
-
-    def test_recipe_doesnt_revise_when_clean(self):
```

Comment:
I think this test is still valid, just without the channels part.

Reply:
Restored, fixed and rebased (new commit message)

Reply:
Maybe there was a different typo that I didn't see, but I think "mcountry" in the commit message isn't right.

Reply:
There. Fixed now.